### PR TITLE
Shaka Packager - Amended as per Perry's comments on the Bento4 docs PR.

### PR DESCRIPTION
**Pull request by bengo1990**
Slight amends also included.

### CODE REVIEW

File: docs/source/DeveloperDocumentation/PackagingGuides/shaka-packager.md [hunk 1]
```diff
@@ -84,11 +82,11 @@ Now you have retrieved the relevant documentation, you can package content with
 
 The values from the CPIX documents will need to be copied and pasted to the corresponding section in the examples below.
 
-For more information, please refer to the Shaka Packager Documentation which can be found at the following link - [Shaka Packager Documentation](https://google.github.io/shaka-packager/html/)
+For more information, please refer to the documentation from [Shaka Packager](https://google.github.io/shaka-packager/html/).
```
Comment (PerritonBranch) [inline docs/source/DeveloperDocumentation/PackagingGuides/shaka-packager.md:85]:
```suggestion
For more information, please refer to the documentation from Google on [Shaka Packager](https://google.github.io/shaka-packager/html/).
```
Think this makes more sense as the docs come from Google and Shaka Packager is the product.

File: docs/source/DeveloperDocumentation/PackagingGuides/shaka-packager.md [hunk 2]
```diff
@@ -84,11 +82,11 @@ Now you have retrieved the relevant documentation, you can package content with
...
 
-### Packaging Content with Widevine and Playready DRM
+### Packaging Content with Widevine and PlayReady DRM
 
-Below is an example of how we can send a request to Shaka Packager to encrypt our content with Widevine and Playready DRM and package it.
+Below is an example of how we can send a request to Shaka Packager to encrypt our content with Widevine and PlayReady DRM and package it.
```
Comment (PerritonBranch) [inline docs/source/DeveloperDocumentation/PackagingGuides/shaka-packager.md:89]:
```suggestion
Below is an example of how we can send a request to Shaka Packager to encrypt and package our content with Widevine and PlayReady DRM.
```
Slight rewording, as the sentence felt like a bit of a mouth full.

File: docs/source/DeveloperDocumentation/PackagingGuides/shaka-packager.md [hunk 3]
```diff
@@ -103,9 +101,9 @@ Please note, the `Widevine_PSSH` from the CPIX Document is in Base64 format. Sha
    --mpd_output /media/<name-of-manifest>.mpd
 ```
 
-### Packaging Content with Fairplay DRM
+### Packaging Content with FairPlay DRM
 
-Below is an example of how we can send a request to Shaka Packager to encrypt our content with Fairplay DRM and package it: 
+Below is an example of how we can send a request to Shaka Packager to encrypt our content with FairPlay DRM and package it:
```
Comment (PerritonBranch) [inline docs/source/DeveloperDocumentation/PackagingGuides/shaka-packager.md:106]:
```suggestion
Below is an example of how we can send a request to Shaka Packager to encrypt and package our content with FairPlay DRM. 
```
Same as above, plus changing `:` to `.` for consistency.

**Review by PerritonBranch — DISMISSED** [review]

File: docs/source/DeveloperDocumentation/PackagingGuides/shaka-packager.md (hunk 2 above)
Comment (bengo1990) [reply docs/source/DeveloperDocumentation/PackagingGuides/shaka-packager.md:89]: Actioned

File: docs/source/DeveloperDocumentation/PackagingGuides/shaka-packager.md (hunk 3 above)
Comment (bengo1990) [reply docs/source/DeveloperDocumentation/PackagingGuides/shaka-packager.md:106]: Actioned.

File: docs/source/DeveloperDocumentation/PackagingGuides/shaka-packager.md (hunk 1 above)
Comment (bengo1990) [reply docs/source/DeveloperDocumentation/PackagingGuides/shaka-packager.md:85]: Actioned.

**Review by PerritonBranch — APPROVED** [review]

**Review by somerobmitchell — APPROVED** [review]

**Pull request merged**